### ISSUE TITLE
Alyx registration error

### ISSUE THREAD

**Issue by alejandropan** [int-brain-lab/ibllib#35]
rc.register_sync(ROOT_DATA_FOLDER)
2019-01-13T01:22:41,601 INFO [extract_session.py:73] Extracting C:\Users\apv2\Documents\IBL\Data\Subjects\ibl_witten_01\2019-01-12\002
2019-01-13T01:22:43,278 ERROR [extract_session.py:80] C:\Users\apv2\Documents\IBL\Data\Subjects\ibl_witten_01\2019-01-12\002 failed extraction
    Unexpected end of string while parsing second. Expected 1 more character
2019-01-13T01:22:43,302 INFO [extract_session.py:73] Extracting C:\Users\apv2\Documents\IBL\Data\Subjects\ibl_witten_02\2019-01-12\001
2019-01-13T01:22:50,276 INFO [extract_session.py:73] Extracting C:\Users\apv2\Documents\IBL\Data\Subjects\ibl_witten_03\2019-01-12\001
Connected to https://alyx.internationalbrainlab.org as alejandro
2019-01-13T01:23:07,201 INFO [registration.py:39] registeringC:\Users\apv2\Documents\IBL\Data\Subjects\ibl_witten_02\2019-01-12\001
2019-01-13T01:23:10,408 WARNING [registration.py:139] No matching dataset type for: C:\Users\apv2\Documents\IBL\Data\Subjects\ibl_witten_02\2019-01-12\001\raw_behavior_data\_iblrig_taskCodeFiles.raw.zip
2019-01-13T01:23:12,126 INFO [registration.py:39] registeringC:\Users\apv2\Documents\IBL\Data\Subjects\ibl_witten_03\2019-01-12\001
2019-01-13T01:23:17,551 WARNING [registration.py:139] No matching dataset type for: C:\Users\apv2\Documents\IBL\Data\Subjects\ibl_witten_03\2019-01-12\001\raw_behavior_data\_iblrig_taskCodeFiles.raw.zip

**Comment by oliche** [comment]
Yes so regarding registration @nbonacchi and I agreed a while ago that `_iblrig_taskCodeFiles.raw.zip` should have been `_iblrig_codeFiles.raw`, it did not yet carry over to the latest version.
I've pushed a fix on develop that will rename  automatically regardless of the rig version.

Another thing we see here is the extraction for `\ibl_witten_01\2019-01-12\002` that fails. I had a look at the data and it seems the wheel positions ssv file is empty. I consider the session is a dud and I don't think we should register it. I've added some loggin functionality on the dev branch but it won't change the behavior.

Basically @alejandropan you're good to go, make sure you pull the latest dev version before extracing/registering the next batch.
